<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="static" default=""/>
        <include 
            file="$(find gazebo_ros)/launch/empty_world.launch">
            <arg name="world_name"  value="$(find vitarana_drone)/worlds/drone$(arg static).world"/>
        </include>
        <!-- <node name="setpoint_control" type="Task_3_VD_2373_setpoint_control.py" pkg="vitarana_drone"/> -->
        <node name="setpoint_selector" type="Task_3_VD_2373_setpoint_selector.py" pkg="vitarana_drone"/>

</launch>